### ISSUE TITLE
twitter, etc icons not displaying correctly on the website

### ISSUE THREAD

**Issue by nicolevasilevsky** [biocuration/isb-website#51]
![image](https://user-images.githubusercontent.com/6722114/43108335-14409f98-8e96-11e8-81f9-03261e198f14.png)

**Comment by andrewsu** [comment]
are you running an adblocker?  If so, do you still see the problem after you disable the adblocker?  That seems to fix it for me, but want to confirm we're seeing the same issue...

**Comment by nicolevasilevsky** [comment]
Yes, I was and turning it off fixed the issue. Looks like the icons changed. Good to know, thanks!

**Comment by andrewsu** [comment]
well, don't want to close this ticket just yet.  It is annoying (and odd) that an adblocker would be blocking these, and there must be a work around.  worth investigating given how many people run adblockers...

**Comment by nicolevasilevsky** [comment]
thanks for investigating!
